wasm/src/circuit_types/keychain.rs

```diff
@@ -230,32 +246,37 @@ pub struct PublicSigningKey {
 
 impl PublicSigningKey {
     /// Construct a key from bytes
-    pub fn from_bytes(bytes: &[u8]) -> Result<Self, String> {
+    pub fn from_bytes(bytes: &[u8]) -> Result<Self, JsError> {
         // Parse the encoded K256 point assumed to be compressed
-        let point = EncodedPoint::from_bytes(bytes).map_err(|e| e.to_string())?;
+        let point = EncodedPoint::from_bytes(bytes).map_err(map_js_error!("{}"))?;
 
         // Convert to circuit-native types, it is simpler to go through the whole
         // conversion than decompressing manually
         let verifying_key =
-            K256VerifyingKey::from_encoded_point(&point).map_err(|e| e.to_string())?;
-        Ok((&verifying_key).into())
+            K256VerifyingKey::from_encoded_point(&point).map_err(map_js_error!("{}"))?;
```

Comment:
Is this shorthand for passing the error message into the map_js_error macro? As in it's not just an empty message?